taipy/gui_core/_context.py

```diff
@@ -203,7 +205,7 @@ def submission_status_callback(self, submission_id: t.Optional[str]):
     def no_change_adapter(self, entity: t.List):
         return entity
 
-    def cycle_adapter(self, cycle: Cycle):
+    def cycle_adapter(self, cycle: Cycle, sorts: t.Optional[t.List[t.Dict[str, t.Any]]] = None):
```

Comment:
Not sure about the name for 'sorts', but nothing better to propose that would be short enough...